bucket/wsa-pacman.json

```diff
@@ -0,0 +1,41 @@
+{
+    "version": "1.1.1",
+    "description": "A GUI package manager and package installer for Windows Subsystem for Android (WSA)",
+    "homepage": "https://github.com/alesimula/wsa_pacman",
+    "license": "GPL-3.0-only",
+    "notes": "For file associations, run '$dir\\install-associations.reg'",
+    "depends": "vcredist2019",
```

Comment:
Is there any documentation for this? The usual practice is to use `suggest` instead of `depends` for vcredist's.

Reply:
https://github.com/alesimula/wsa_pacman/issues/10#issuecomment-1000503332

Reply:
I'd recommend using 'suggest' field.